docs/app/components/navigation-bar-search/navigation-bar-search.component.ts

```diff
@@ -32,7 +33,21 @@ export class NavigationBarSearchComponent {
     private data: ISearchResult[];
     private history: ISearchResult[] = [];
 
-    constructor(private router: Router, private navigation: NavigationService) {
+    private angular = true;
+
+    private ngVersions = Version;
```

Comment:
Remove this line. We only need to do this if we want to expose the `Version` enum to the template. When using it in a class you can just do `Version.AngularJS` etc..